lib/features/composer/presentation/composer_controller.dart

```diff
@@ -802,18 +820,22 @@ class ComposerController extends BaseController with DragDropFileMixin implement
         listToEmailAddress = List.from(recipients.value1.toSet());
         listCcEmailAddress = List.from(recipients.value2.toSet());
         listBccEmailAddress = List.from(recipients.value3.toSet());
+        listReplyToEmailAddress = List.empty(); //empty ? not sure about that
+        listReplyToEmailAddress = List.empty(); //empty ? not sure about that
```

Comment:
remove duplicated

Reply:
why is it empty?

Reply:
i left it empty at first because when composing a new mail, you would want the field `replyTo` to be initially empty. but digging further, there are test cases where it should not be empty: for example when editing a draft mail for which the `replyTo` field had been filled.
this is now fixed.